### ISSUE TITLE
Dask notebook, Colab Day_5 - error on slider

### ISSUE THREAD

**Issue by pwalczysko** [ome/EMBL-EBI-imaging-course-05-2023#8]
This error only appears when the slider is moved on the widget under the cell

```
import matplotlib.pyplot as plt
%matplotlib inline
from ipywidgets import *

def display(i=0):
    r, name = results[i]
    fig = plt.figure(figsize=(10, 10))
    plt.subplot(121)
    plt.imshow(r)
    plt.title(name)
    fig.canvas.flush_events()

interact(display, i= widgets.IntSlider(value=0, min=0, max=len(results)-1, step=1, description="Select Plane", continuous_update=False))
```

Error:
```
TypeError                                 Traceback (most recent call last)

[/usr/local/lib/python3.10/dist-packages/ipywidgets/widgets/interaction.py](https://localhost:8080/#) in update(self, *args)
    255                     value = widget.get_interact_value()
    256                     self.kwargs[widget._kwarg] = value
--> 257                 self.result = self.f(**self.kwargs)
    258                 show_inline_matplotlib_plots()
    259                 if self.auto_display and self.result is not None:

[<ipython-input-17-bb0190ad256f>](https://localhost:8080/#) in display(i)
      4 
      5 def display(i=0):
----> 6     r, name = results[i]
      7     fig = plt.figure(figsize=(10, 10))
      8     plt.subplot(121)

TypeError: cannot unpack non-iterable int object
```

**Comment by jburel** [comment]
I will push a fix

**Comment by jburel** [comment]
I ran the notebook in Colab and I did not have any issue

**Comment by pwalczysko** [comment]
> I ran the notebook in Colab and I did not have any issue

Yes, if you do not slide the slider, there is no issue. You have to move the slider on the image to elicit the error.

**Comment by jburel** [comment]
I moved the slider and it worked as expected

**Comment by pwalczysko** [comment]
I am still having today with fresh re-run

![Screenshot 2023-05-19 at 11 48 51](https://github.com/ome/EMBL-EBI-imaging-course-05-2023/assets/2478303/00b42a50-0386-49c6-a207-14e53eeeb2ca)

**Comment by pwalczysko** [comment]
I think the mystery is solved.
the problem seems to be that I am using the "Run all cells" button in Colab.
I think that the idea is that the cells do get run sequentially, but somehow it does not always work ?

The way I "fixed" the problem was to re-run the computation cell above, 

``%time results = dask.compute(*lazy_results)``

manually clicking on the run button next to that cell. Then, I manually clicked onto each next cell. 

This changed the output to be a bit different on the ``print`` cell which we added and also, it removed the error. Now, even if I slide the slider, it just plots a new image - just like in @jburel 's testing - see screenshot below.


![Screenshot 2023-05-19 at 12 01 24](https://github.com/ome/EMBL-EBI-imaging-course-05-2023/assets/2478303/446f51a1-98ae-4dc7-b52b-9181b23ee898)

![Screenshot 2023-05-19 at 12 13 17](https://github.com/ome/EMBL-EBI-imaging-course-05-2023/assets/2478303/298d4eec-cd94-4968-928d-6914a91f81c1)
